### PR TITLE
Reduce image steps

### PR DESCRIPTION
**Pull request by k88hudson**
This will start up the image/gallery picker right away when a user adds an image to a page.

### CODE REVIEW

**Comment by k88hudson** [comment]
Requires https://github.com/mozilla/webmaker-android/pull/2436 in webmaker-android

File: src/components/get-media/get-media.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
+var React = require('react');
+var classNames = require('classnames');
+
+// window.Platform = {};
```
Comment (k88hudson) [inline src/components/get-media/get-media.jsx]: you can use this to simulate in the browser if you want

**Comment by vazquez** [comment]
awesome

**Comment by vazquez** [comment]
Works like a charm!! :star:

**Comment by k88hudson** [comment]
@vazquez you are the best :dancer:

**Pull request merged**